spec/controllers/hq/orphans_controller_spec.rb

```diff
@@ -10,8 +10,65 @@
 
   specify '#show' do
     expect(Orphan).to receive(:find).and_return(orphan)
+
     get :show, id: orphan.id
+
     expect(assigns(:orphan)).to eq orphan
     expect(response).to render_template 'show'
   end
+
+  context 'new and #create' do
+    let(:orphan) { build_stubbed :orphan }
+
+    before :each do
+      controller.instance_variable_set(:@orphan, orphan)
+    end
+
+    specify 'successful create redirects to the show view' do
+      expect(orphan).to receive(:save).and_return(true)
+
+      post :create, orphan: orphan.attributes
+
+      expect(response).to redirect_to hq_orphan_path(orphan)
+    end
+
+    specify 'unsuccessful create renders the new view' do
+      expect(orphan).to receive(:save).and_return(false)
+
+      post :create, orphan: orphan.attributes
+
+      expect(response).to render_template 'new'
+    end
+  end
+
+  context '#edit and #update' do
+    before :each do
+      @orphan = build_stubbed :orphan
```

Comment:
It would be best to use a consistent approach regarding using `let` vs. instance variables, and testing or not whether the instance variable is set by the controller action (L16 vs. here).